Machine Learning Model/Model Design and Testing/Provisional Machine Learning Model.ipynb

```diff
@@ -0,0 +1,744 @@
+       "      <th></th>\n",
+       "      <th>Column 1</th>\n",
+       "      <th>Column 2</th>\n",
+       "      <th>Column 3</th>\n",
+       "      <th>Column 4</th>\n",
+       "      <th>Column 5</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>7538</td>\n",
+       "      <td>166.266051</td>\n",
+       "      <td>0.230211</td>\n",
+       "      <td>33371.825710</td>\n",
+       "      <td>43</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2007</td>\n",
+       "      <td>323.146614</td>\n",
+       "      <td>0.772611</td>\n",
+       "      <td>10529.291222</td>\n",
+       "      <td>48</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2364</td>\n",
+       "      <td>348.914007</td>\n",
+       "      <td>0.225228</td>\n",
+       "      <td>39763.582434</td>\n",
+       "      <td>21</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4525</td>\n",
+       "      <td>255.428136</td>\n",
+       "      <td>0.483272</td>\n",
+       "      <td>3689.768227</td>\n",
+       "      <td>8</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>9488</td>\n",
+       "      <td>850.711051</td>\n",
+       "      <td>0.913506</td>\n",
+       "      <td>8742.182866</td>\n",
+       "      <td>40</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Column 1    Column 2  Column 3      Column 4  Column 5\n",
+       "0      7538  166.266051  0.230211  33371.825710        43\n",
+       "1      2007  323.146614  0.772611  10529.291222        48\n",
+       "2      2364  348.914007  0.225228  39763.582434        21\n",
+       "3      4525  255.428136  0.483272   3689.768227         8\n",
+       "4      9488  850.711051  0.913506   8742.182866        40"
+      ]
+     },
+     "execution_count": 203,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Create new data for the model to make predictions with \n",
+    "# Create Random Data and place into Df\n",
+    "# Create Random Data\n",
+    "col_1 = np.random.randint(0, 10001, (501,1), dtype='int64')\n",
+    "col_2 = np.random.rand(501,1)*1000\n",
+    "col_3= np.random.rand(501,1)\n",
+    "col_4= np.random.rand(501,1)*50000\n",
+    "col_5 = np.random.randint(0, 86, (501,1), dtype='int64')\n",
+    "data = [col_1, col_2, col_3, col_4, col_5]\n",
+    "columns = ['Column 1', 'Column 2', 'Column 3', 'Column 4', 'Column 5']\n",
+    "# Create Random Data df\n",
+    "newdata_df=pd.DataFrame(index=list(range(0,501)))\n",
+    "for i in list(range(0,5)):\n",
+    "    newdata_df[columns[i]]=data[i]\n",
+    "newdata_df.head()"
```

Comment:
When we meet again can you explain this part to me, just because I'm not super strong on these random functions.